2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1186,75 +1186,97 @@ _Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.3.8_
-  + il Verificatore esprime un giudizio e segnala eventuali problematiche riscontrate.
+  + il Verificatore deve valutare i prodotti e i servizi al fine di garantirne la conformità rispetto ai criteri stabiliti;
+  + il Verificatore deve assicurarsi che la verifica e la convalida degli output dei processi del ciclo di vita siano eseguiti conformemente con quanto concordato in precedenza;
+  + il Verificatore deve applicare il processo di misurazione della qualità per verificare che il prodotto o servizio rispetti le metriche precedentemente stabilite;
+  + il Verificatore deve esprimere un giudizio e segnalare eventuali problematiche riscontrate.
 
-+ gestire rapporti del controllo della qualità:
++ *gestire report e record del Controllo della Qualità*:
 
- - essi consistono in:
+ - *l'attività consiste in*:
 
-  + stilare rapporti relativi alle attività di controllo della qualità:
-    - in sede di retrospettiva il Verificatore aggiorna il gruppo riguardo le attività di controllo qualità dello sprint in corso;
-    - redazione di un verbale contenente anche gli esiti del processo di controllo della qualità.
+  + *stilare report e record relativi alle attività di Controllo della Qualità*:
+    - i report e i record vengono generati tramite l'utilizzo coordinato di Jira, Google Sheets e Grafana che porta alla creazione di un cruscotto di qualità. Si rimanda a @tecnologie_controllo per ulteriori dettagli.
 
-  + mantenere, archiviare e distribuire i rapporti;
+  + *mantenere, archiviare e distribuire i report*:
+    - il cruscotto di qualità è consultabile in ogni momento da ogni membro del gruppo, che possiede un link per accedervi. L'aggiornamento del cruscotto avviene in automatico e in tempo reale grazie al suo collegamento con Jira.
 
-  + identificare incidenti e problemi associati alle valutazioni effettuate.
+  + *identificare incidenti e problemi associati alle valutazioni effettuate*:
+    - in sede di retrospettiva, il gruppo lavora in modo coordinato per identificare gli incidenti e i problemi, servendosi delle informazioni presentate dal Verificatore tramite il cruscotto di qualità;
+    - l'Amministratore si occupa della redazione di un verbale contenente anche gli esiti del processo di Controllo della Qualità.
 
-+ trattare incidenti e problemi:
+  + *trattare incidenti e problemi*:
 
-  + in caso di segnalazione di incidenti e problemi viene svolto un lavoro collettivo tra Verificatore e redattore per la loro risoluzione;
-  + nel caso tali problemi o incidenti siano di carattere generale, viene avvisato collettivamente il gruppo dell'insorgenza di tali problemi o incidenti, al fine di evitare future ricorrenze degli stessi.
+    + in caso di segnalazione di incidenti e problemi deve essere svolto un lavoro collettivo per la loro risoluzione;
+    + tutte le criticità devono prevedere risoluzioni e arginamenti già predisposti all'interno del documento dell'_Analisi dei Rischi_.
```

Comment:
manca il riferimento alla versione del documento dell'analisi dei rischi

Reply:
ho aggiornato la versione, era indicata la 2.0.0 ma in repo abbiamo solo la 1.0.0